### DISCUSSION TITLE
Delete CRON

### DISCUSSION THREAD

**Discussion by mesfandi** [elsa-workflows/elsa-core#2550] [Q&A]
Hi,
Is it possible to see the timer and cron tasks? is there anywhere on the console or on the database that I can list and see them?  
I had a CRON and published it several times, so now it runs several instances at the same time. I tried to delete them by clear timer on the designer, however it did not work.

**Comment by mesfandi** [comment]
Hi, 
I have another question. I realized that the CRON activities sometimes are executed several times. I think this is because my azure app service has a scale out feature and it autoscale the service to several instances and each instance would run the crone separately. 
For example in this picture, I defined a CRON of every 15 minutes after 35 minute of every hour. It is executed one instance on 10:35 and on 10:50. However, in 11:35 it is executed three times.  
![image](https://user-images.githubusercontent.com/40899452/144468250-c8cd80fb-930e-4f1b-bcfd-c48165b65744.png)

**Comment by sfmskywalker** [comment]
If you configure the temporal activity provider with DB persistence, then you should be able to see the jobs in the corresponding database tables. If you're using Quartz.NET, checkout their documentation to learn about the database tables to look for. Same goes for Hangfire.

> I had a CRON and published it several times, so now it runs several instances at the same time. I tried to delete them by clear timer on the designer, however it did not work.

To clear the timers, you should delete the workflow instances. Clearing the timer from the designer won't work, because the workflow definition will simply be saved as a new version.

> I realized that the CRON activities sometimes are executed several times.

Make sure that you configure the temporal activity provider (Quartz.NET or Hangfire) to use persistent storage so that they can synchronize between nodes. You can find more information about this [here](https://elsa-workflows.github.io/elsa-core/docs/next/hosting/hosting-distributed-hosting#distributed-temporal-services).
